services/npm/npm-rating.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const Joi = require('joi')
+const { starRating } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+// https://api-docs.npms.io/#api-Package-GetPackageInfo
+const responseSchema = Joi.object({
+  score: Joi.object({
+    final: Joi.number().required(),
+    detail: Joi.object({
+      maintenance: Joi.number().required(),
+      popularity: Joi.number().required(),
+      quality: Joi.number().required(),
```

Comment:
We do have various schema helpers for common functions, but seems these will be unsigned decimal values, where we often do this to be very explicit:
```suggestion
    final: Joi.number().min(0).required(),
    detail: Joi.object({
      maintenance: Joi.number().min(0).required(),
      popularity: Joi.number().min(0).required(),
      quality: Joi.number().min(0).required(),
```